src/alchemlyb/parsing/namd.py

```diff
@@ -12,13 +12,13 @@
 k_b = 1.9872041e-3
 
 
-def extract_u_nk(fep_file, T):
-    """Return reduced potentials `u_nk` from NAMD fepout file.
+def extract_u_nk(fep_files, T):
+    """Return reduced potentials `u_nk` from NAMD fepout file(s).
```

Comment:
Document that IDWS is now supported (reference to paper/NAMD docs?).

Document that parsing IDWS file can discard data.

Explain the use of the multiple `fep_files` as input. I am not familiar with NAMD's FEP code and output so I have no idea if this is obvious or not. A simple example might help and really _any_ docs are better than none.

Reply:
Please also add a 
```reST


.. versionchanged:: 0.5.0
   Support for Interleaved Double-Wide Sampling files added. 
   `fep_files` can now be a list of filenames.
```

Reply:
Please add the versionchanged to the docs, just updated to **0.6.0**.

Reply:
I am adding it.